boundary_layer_default_plugin/preprocessors.py

```diff
@@ -110,6 +110,14 @@ def process_arg(self, arg, node, raw_args):
                 regex.pattern)
             return arg
 
+        VERBATIM_REGEX = '<<item_name>>|<<item.name>>|<<item.value>>'
```

Comment:
Hey so one feature that I would like to support is the ability to actually _require_ that the argument be a (or contain a) verbatim string...  For example, there are some operators that take methods as arguments, and the only way we can support those right now is for the parameter passed to the operator to be a lambda (at the moment we do not enforce any restrictions on these, but we probably should).

So could we do the `VERBATIM_REGEX` check only if the `regex.match()` check fails?  Maybe change the code to something like (starting at line 114 and replacing all of the rest of the method):

```python
if regex.match(rendered_arg):
    return arg

VERBATIM_REGEX = ...
if re.compile(VERBATIM_REGEX).search(rendered_arg):
    logger.warning(...)
    return arg

raise Exception('Invalid argument...')
```

Reply:
Oh and also, I think we can loosen the definition of the `VERBATIM_STRING` regex to something like `<<.+>>`, because generally speaking it's possible for any verbatim string to be present, and we can't check any of them.

Reply:
Thanks for the comments. Done!